web/styles/popovers.css

```diff
@@ -1492,3 +1472,7 @@ ul.navbar-dropdown-menu-outer-list {
         outline-offset: -1px;
     }
 }
+
+#message-actions-menu-dropdown {
+    --popover-menu-max-width: 350px;
```

Comment:
Why set this variable inside this ID? Would it make better sense to offer this dimension to other popover menus? And keep it with `app_variables.css`?

Reply:
To explain the logic behind this, I would need to explain the code snippet from my commit below:

```css
    .simplebar-content {
        /* Set max-width to the value defined by --popover-menu-max-width
        custom property, else fallback to 97vw. */
        max-width: min(var(--popover-menu-max-width, 97vw), 97vw);
    }
```

Here, the intended behavior is this: "We only want a max-width on popovers where we explicitly define it, else we want it to be responsive with the `97vw` value".

In the case that the `--popover-menu-max-width` custom property is not defined (default case), the fallback value of `97vw` is considered, and we get a reduced form `max-width: min(97vw, 97vw); === max-width: 97vw;`.

When we then set the `--popover-menu-max-width` custom property for a certain popover (via targeting its `ID` in this case), we get the reduced form of `max-width: min(--popover-menu-max-width, 97vw);` where `--popover-menu-max-width` comes into play at the desktop sizes, and the `97vw` value comes into play at mobile sizes.

Reply:
Please let me know, if you would prefer an alternative to this behavior, thanks!

Reply:
This approach seems OK, but let's make sure to document the motivation for the localized variable in a comment.

Reply:
```css
/* Set `max-width` to the minimum of `97vw` and the value defined by
the `--popover-menu-max-width` custom property, which should be defined
as a local custom property in the scope of a particular popover where
we want apply a `max-width` limit.

When we don't define this local custom property, the fallback value of
`97vw` is considered and the `max-width` is then set to `min(97vw, 97vw)`
which is simply equivalent to `97vw`.

IMPORTANT: Setting `--popover-menu-max-width` in the `:root` scope
instead of the local scope, eliminates the fallback behavior and thus
prevents us from setting the max-width on a case-to-case basis. */
max-width: min(var(--popover-menu-max-width, 97vw), 97vw);
```

This is the comment I came up with, to document the motivation for the localized variable.